exercises/introducao-a-javascript-e-logica-de-programacao_1/dia_1/exercicio_7.js

```diff
@@ -0,0 +1,23 @@
+let nota = 86;
+
+if (nota >= 90) {
+  console.log("A");
```

Comment:
Nesse caso, caso o número seja maior que 100 ele vai retornar A antes mesmo de chegar na mensagem de erro que tá lá embaixo! Uma solução é trazer a mensagem de erro pra ser analisada primeiro, ou você pode simplesmente adicionar uma nova condição à linha 3 pra só analisar se for menor ou igual a 100!

Nesse último caso você teria que atualizar todos os ifs pra colocar um teto, então eu acho que seria mais fácil  trazer a checagem do erro pra cima mesmo.